build/scripts/build.ps1

```diff
@@ -587,6 +587,10 @@ try {
     # VSIX, NuGet doesn't support re-packing hence we have to order it this way.
     if ($pack) {
         Build-NuGetPackages
+
+        if ($cibuild -or $official) { 
+            Build-DeployToSymStore
```

Comment:
>Build-DeployToSymStore [](start = 12, length = 22)

Why do we want to do this for cibuild? The only leg which currently creates our NuGet packages in the Microbuild leg. We don't run tests there. Hence not sure why we would deploy symbols

Reply:
I was thinking it would be better to run on CI in case the converter failed. That means there is a bug in the compiler (producing bad PDBs) or the converter tool. It might be rare enough to only enable in microbuild. I can remove it if you don't think that's useful.

Reply:
That makes sense. I agree we should keep it.